packages/ckeditor-cloud-services-core/tests/token/token.js

```diff
@@ -202,49 +215,39 @@ describe( 'Token', () => {
 		} );
 	} );
 
-	describe( '_startRefreshing()', () => {
-		it( 'should start refreshing', () => {
-			const clock = sinon.useFakeTimers( { toFake: [ 'setInterval' ] } );
+	describe( '_registerRefreshTokenTimeout()', () => {
```

Comment:
Should we test private methods? :thinking:

Reply:
Tests for private methods were removed.